src/Sql/PostgreSQL/Schema/bitwarden.sql

```diff
@@ -0,0 +1,3 @@
+DROP SCHEMA bitwarden;
+
+CREATE SCHEMA bitwarden AUTHORIZATION bitwarden;
```

Comment:
no, you can just use the default public schema, but it would probably mean it would always have to use a dedicated database with no option to use an existing postgreSQL database with a separate schema.  Some installs might prefer to share the database, so it makes sense to just build it to accommodate both to begin with.

Reply:
If dapper can access [dbo].table in sql server, then it can use a postgreSQL schema no issues. It's basic functionally.  

Using the public schema in postgreSQL is bit like using the master database in SQL Server. You can do it, but should you?  

In any case, I can change it to suit. There is still a lot of translation to go before it can even be tested.I also only really know SQL. I'm not a developer that knows how to program in dapper

Reply:
Public has roles/users, system information, system functions, extensions, security rules , table space allocations, permissions, much like the master database.  

It also does the same thing as dbo that's true, but so does a different schema with a user that isn't a super user. 

if someone already has a postgreSQL instance, what would be the reason to deny the use of a differently named schema in an existing database?  Isn't one of the reasons for the original request a much smaller footprint memory wise. Taking away the schema limits that.

I can use public if it's easierfor you, I'd rather see a working concept that can be improved later

Reply:
You can set search_path parameter in postgreSQL for a user so that you can still use unqualified names and the objects will still reside in the correct schema. I'll add that in today sometime

Reply:
added a commit with all these discussed changes.

if the bitwarden.sql script isn't run, then objects will go into public, if it is run, objects will go to the bitwarden schema, however the search_path parameter is modified for the bitwarden role, so code doesn't have to be change to access a particular schema qualified name. 

i've changed all the other sql to reflect the changes, and converted to snake_case, and functions only